devices/lavender/initramfs/init

```diff
@@ -1,33 +1,43 @@
-#!/bin/sh
+#!/sbin/busybox sh
```

Comment:
This script would run inside the initramfs, so it depends on busybox's `sh`. I figured it is easier to call busybox directly rather than symlinking all its utilities and then calling the symlinks.